stdlib/ballerina-mime/src/main/ballerina/ballerina/mime/natives.bal

```diff
@@ -105,6 +105,11 @@ public native function <Entity entity> setByteChannel (io:ByteChannel byteChanne
 @Return {value:"Return a byte channel"}
 public native function <Entity entity> getByteChannel () (io:ByteChannel);
 
+@Description {value:"Given an entity, get it's body parts."}
+@Param {value:"entity: Represent a mime Entity"}
```

Comment:
shouldn't this be `MIME` instead of `mime`? 

[https://en.wikipedia.org/wiki/MIME](https://en.wikipedia.org/wiki/MIME)